api/v1alpha1/perconaservermysql_types.go

```diff
@@ -200,6 +209,12 @@ type BackupStorageAzureSpec struct {
 	StorageClass string `json:"storageClass,omitempty"`
 }
 
+type MySQLRouterSpec struct {
+	Expose ServiceExposeTogglable `json:"expose,omitempty"`
```

Comment:
we do not need enable option for router

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/91/commits/862d03e22c696064b7de939b4948503e61b5f6bf